### ISSUE TITLE
[Accessibility]The WAI-ARIA property is not supported global property on elements that does not have a WAI-ARIA role.

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/kimchi#904]
In order to use any WAI-ARIA property/states in any element,we need to assign WAI-ARIA role to the element.
